src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -148,14 +148,15 @@ public function onSubmit(FormEvent $event)
             throw new UnexpectedTypeException($data, 'array or (\Traversable and \ArrayAccess)');
         }
 
-        if ($this->deleteEmpty) {
-            $previousData = $event->getForm()->getData();
+        if ($entryFilter = $this->deleteEmpty) {
```

Comment:
because `$this->deleteEmpty()` will call undefined method `ResizeFormListener::deleteEmpty()`